2025-10-28 08:03:05,605 : INFO : root : Loading config.
2025-10-28 08:03:05,609 : INFO : YamlMultifileConfigLoader : Machine config file #1: config.yaml
2025-10-28 08:03:05,610 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\fdca14d0c9d658ab8200d57120800af5.mpf_cache
2025-10-28 08:03:05,764 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\5dec96b1547c4adf5bdd3d1715e74bc0.mpf_cache
2025-10-28 08:03:05,765 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\202a5840610e95051f9f473dfa660cc0.mpf_cache
2025-10-28 08:03:05,765 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\5dec96b1547c4adf5bdd3d1715e74bc0.mpf_cache
2025-10-28 08:03:05,766 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\2cd13a2d3f375e00ab01d6d09a85d8e7.mpf_cache
2025-10-28 08:03:05,767 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\9220b44b97b91972524500b81035d790.mpf_cache
2025-10-28 08:03:05,768 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\31ee881584fa6fce665a672a4f3f633a.mpf_cache
2025-10-28 08:03:05,769 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\5e841763ec73085bbf92358fe6afda21.mpf_cache
2025-10-28 08:03:05,769 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\7f73d12aa9beccac1cbd8c265ccda72e.mpf_cache
2025-10-28 08:03:05,770 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\b17152efa55a24f0424602723f8d3921.mpf_cache
2025-10-28 08:03:05,770 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\e87fc0e7d2f8a851b5fa3f604e0d0c6e.mpf_cache
2025-10-28 08:03:05,771 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\196ac76b6cab3a9880804c826cfbcb5c.mpf_cache
2025-10-28 08:03:05,771 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\834292dd7853e12a66732dc89828c828.mpf_cache
2025-10-28 08:03:05,772 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\ae2be9c5579852c305357e207df97e8b.mpf_cache
2025-10-28 08:03:05,773 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\46f8fa3e41dfeeb8232af2f03a13b6c4.mpf_cache
2025-10-28 08:03:05,773 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\c67e9c175da6c060302945929b27b878.mpf_cache
2025-10-28 08:03:05,774 : INFO : Machine : Mission Pinball Framework Core Engine v0.80.0.dev11
2025-10-28 08:03:05,774 : INFO : Machine : Command line arguments: {'no_load_cache': False, 'create_config_cache': True, 'bcp': True, 'configfile': ['config.yaml'], 'force_assets_load': False, 'platform_integration_test': False, 'jsonlogging': False, 'logfile': 'logs\\2025-10-28-08-03-05-mpf-LAPTOP-MPGP6R69.log', 'pause': False, 'production': False, 'text_ui': True, 'loglevel': 15, 'consoleloglevel': 20, 'force_platform': None, 'syslog_address': None, 'full_logfile_path': 'C:\\Users\\antho\\nightpin\\logs\\2025-10-28-08-03-05-mpf-LAPTOP-MPGP6R69.log'}
2025-10-28 08:03:05,774 : INFO : Machine : MPF path: C:\Users\antho\mpf\Lib\site-packages\mpf
2025-10-28 08:03:05,774 : INFO : Machine : Machine path: C:\Users\antho\nightpin
2025-10-28 08:03:05,774 : INFO : Machine : Platform: win32
2025-10-28 08:03:05,774 : INFO : Machine : Python executable location: C:\Users\antho\mpf\Scripts\python.exe
2025-10-28 08:03:05,774 : INFO : Machine : Python version: 3.12.7 (64-bit)
2025-10-28 08:03:05,777 : INFO : Machine : Initializing MPF...
2025-10-28 08:03:06,015 : INFO : lisy : Connecting to com3 at 115200bps
2025-10-28 08:03:06,016 : WARNING : lisy : Could not import terminos (this is ok on windows).
2025-10-28 08:03:06,019 : ERROR : asyncio : Fatal write error on serial transport
protocol: <asyncio.streams.StreamReaderProtocol object at 0x000001E35F0E93A0>
transport: SerialTransport(<ProactorEventLoop running=True closed=False debug=False>, <asyncio.streams.StreamReaderProtocol object at 0x000001E35F0E93A0>, Serial<id=0x1e35f3a87c0, open=False>(port='com3', baudrate=115200, bytesize=8, parity='N', stopbits=1, timeout=0, xonxoff=False, rtscts=False, dsrdtr=False))
Traceback (most recent call last):
  File "C:\Users\antho\mpf\Lib\site-packages\serial_asyncio\__init__.py", line 290, in _poll_read
    if self.serial.in_waiting:
       ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\serial\serialwin32.py", line 259, in in_waiting
    raise SerialException("ClearCommError failed ({!r})".format(ctypes.WinError()))
serial.serialutil.SerialException: ClearCommError failed (OSError(9, 'The handle is invalid.', None, 6))
2025-10-28 08:03:06,059 : INFO : Machine : Shutting down...
2025-10-28 08:03:06,059 : INFO : EventManager : Event: ======'shutdown'====== Args={}
2025-10-28 08:03:06,059 : ERROR : root : ClearCommError failed (OSError(9, 'The handle is invalid.', None, 6))
Traceback (most recent call last):
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\commands\game.py", line 229, in __init__
    self.machine.run()
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\machine.py", line 698, in run
    if not self.initialize_mpf():
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\machine.py", line 686, in initialize_mpf
    self._crash_shutdown()
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\machine.py", line 751, in _crash_shutdown
    self._do_stop()
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\machine.py", line 742, in _do_stop
    self.shutdown()
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\machine.py", line 772, in shutdown
    self._platform_stop()
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\machine.py", line 822, in _platform_stop
    hardware_platform.stop()
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\platforms\lisy\lisy.py", line 565, in stop
    self.machine.clock.loop.run_until_complete(self._writer.wait_closed())
  File "C:\Users\antho\AppData\Local\Programs\Python\Python312\Lib\asyncio\base_events.py", line 687, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\antho\AppData\Local\Programs\Python\Python312\Lib\asyncio\streams.py", line 364, in wait_closed
    await self._protocol._get_close_waiter(self)
  File "C:\Users\antho\mpf\Lib\site-packages\serial_asyncio\__init__.py", line 290, in _poll_read
    if self.serial.in_waiting:
       ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\serial\serialwin32.py", line 259, in in_waiting
    raise SerialException("ClearCommError failed ({!r})".format(ctypes.WinError()))
serial.serialutil.SerialException: ClearCommError failed (OSError(9, 'The handle is invalid.', None, 6))
